iosHackthonApp/Modules/Features/Login/Sources/Login/LoginViewModel.swift

```diff
@@ -31,9 +31,10 @@ public final class LoginViewModel: ObservableObject {
 	}
 
 	func login(email: String, password: String) {
-		//TODO: API key should be in the shared layer
+		//TODO: API key to be handled in shared layer
+		// To run the project for now, add the api key and run
 		isLoading = true
-		firebase.signIn(apiKey: "AIzaSyCXmrUtOgzc4kj8aimSkmjOcCV9PR438-o", email: email, password: password, returnSecureToken: true, completionHandler: { result, error in
+		firebase.signIn(apiKey: "", email: email, password: password, returnSecureToken: true, completionHandler: { result, error in
```

Comment:
hmm do we even need the apiKey parameter now? Or can it at least be nullable?

Reply:
For now I think we need it as a placeholder, just to run the project. 
We have another ticket to handle the API Key storage, once that is implemented we can get rid of this from here.
Added a TODO for this as well.